packages/block-editor/src/components/image-size-control/index.js

```diff
@@ -43,10 +45,11 @@ export default function ImageSizeControl( {
 		<>
 			{ ! isEmpty( imageSizeOptions ) && (
 				<SelectControl
-					label={ __( 'Image size' ) }
+					label={ label === undefined ? __( 'Image size' ) : label }
 					value={ slug }
 					options={ imageSizeOptions }
 					onChange={ onChangeImage }
+					help={ help }
```

Comment:
I'm not sure if we should add a new property here instead of adding a specific message. I mean `ImageSizeControl` has more controls and providing a `help` property that is going to be used specifically for one of them might be confusing. If we have a good help message here we might not even need label as a prop too 🤔 .

--cc @ciampo @mirka

Reply:
There's a few differences between the `label` and the `help` props for the `SelectControl` component:

- `label` is displayed on top of the dropdown, and is also used to _label_ the `select` input for assistive technology purposes. The label can also be hidden visually through the `hideLabelFromVision` prop if needed (it will still be used by assistive technology, though)
- `help` is displayed with smaller font below the dropdown, and is also used to _describe_ the `select` input for assistive technology purposes

These combinations can be experimented with in [Storybook](https://wordpress.github.io/gutenberg/?path=/story/components-selectcontrol--with-label-and-help-text)

![Screenshot 2022-05-19 at 12 42 21](https://user-images.githubusercontent.com/1083581/169275424-0494623c-764a-4d94-abcb-b43544245a6b.png)

My 2 cents:
- given its importance for assistive technologies, the `label` prop should ideally be always defined. If need be, it can also be hidden for aesthetic reasons by using `hideLabelFromVision`.
- `help` text may be added or not on top of `label` — that decision feels more like a design/UX decision which I don't feel like I'm a good spot to make

Reply:
I agree that if there is a good enough help text we might not need to customize the label.

But I think it will be difficult to write a help text that is fit for the different usages. I am not sure if that is what you meant.

Reply:
That's what I meant yes, but I see your point. Maybe a more specific name of the props to indicate in which control are going to be used?

Reply:
I am not sure I understood, you mean like calling it "imageSizeHelp"?

Reply:
Something like that, yes.